fsp_email.py

```diff
@@ -87,6 +87,22 @@ def parse_date(tz, m):
         raise UnusableEmail("Unable to convert %s to UTC: %r" % (local, e))
     return z.replace(tzinfo=None)
 
+def monthToNum(shortMonth):
```

Comment:
Follow PEP-8 naming conventions for [function names](https://www.python.org/dev/peps/pep-0008/#function-names) and for variable names. This variant of CamelCase with inconsistent case but the first letter is horrible and unreadable (my opinion, apparently) and non-pythonic (*not* my opinion). These should be `month_to_num` and `short_month`.